src/Game.tsx

```diff
@@ -62,32 +48,33 @@ export function Game({ names }: Props): JSX.Element {
   }, []);
 
   return (
-    <div>
+    <div className="page-wrapper">
       <Link to="/">
-        {''}
         <button className="button-close-game">Close game</button>
       </Link>
       <h1 className="header-text">Pokemons ping-pong</h1>
-      {names.length === 2 ? (
-        <div ref={setWrapperElem} className="game-wrapper">
-          <div ref={setFieldDivElem} className="field">
-            <div className="dotted-line"></div>
-            <Ball x={ballCoords.x} y={ballCoords.y} />
-          </div>
-          <PlayerCard
-            y={player1Coords.y}
-            x={player1Coords.x}
-            name={names[0]}
-          ></PlayerCard>
-          <PlayerCard
-            y={player2Coords.y}
-            x={player2Coords.x}
-            name={names[1]}
-          ></PlayerCard>
+
+      {pokemonNames.includes('') && <Redirect to="/" />}
```

Comment:
Empty pokemon names should never be passed to this component. No need for this check.

Reply:
This check is needed in case the user immediately goes to the game page by url /game, skipping the main page